admin/kernel-syncdata-service/src/test/java/io/mosip/kernel/syncdata/test/service/SyncMasterDataServiceHelperTest.java

```diff
@@ -299,4 +311,322 @@ public void getClientSettingsJsonFileTest3() throws Exception {
         Assert.assertNotNull(responseEntity.getHeaders().get("file-signature"));
         Assert.assertNotNull(responseEntity.getHeaders().get(HttpHeaders.CONTENT_DISPOSITION));
     }
+
+    @Test
+    public void convertTemplateEntityToDtoTest01(){
+
```

Comment:
Why you are giving 1 extra line space after method definition. Please remove the line space

Reply:
Removed